app/uk/gov/hmrc/hmrcemailrenderer/templates/eeitt/apdRegistrationConfirmation.scala.txt

```diff
@@ -0,0 +1,23 @@
+@(params: Map[String, Any])
+
+HMRC received your Air Passenger Duty registration
```

Comment:
Please remove this - this is the subject

Reply:
so would you like me to remove both lines: 
@(params: Map[String, Any])

and

@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, "HMRC received your Air Passenger Duty registration"){